package.json

```diff
@@ -11,19 +11,24 @@
     "now-start": "PRIVATE_KEY=$(echo $PRIVATE_KEY_BASE64 | base64 -d) npm start",
     "test": "jest",
     "test:watch": "jest --watch --notify --notifyMode=change --coverage",
+    "lint": "eslint ./lib",
```

Comment:
no list of `index.js`?

Reply:
Ah good catch! I'll probably change it to lint everything except for node_modules

Reply:
Fixed in #160